r6sapi/operators.py

```diff
@@ -125,7 +125,11 @@
     "YING": "dazzler_gadget_detonate",
     "DOKKAEBI": "phoneshacked",
     "VIGIL": "diminishedrealitymode",
-    "ZOFIA": "concussiongrenade_detonate"
+    "ZOFIA": "concussiongrenade_detonate",
+    "WARDEN": "Observation Tool Deceived",
+    "NAKK": "Observation Tool Deceived",
```

Comment:
The ubisoft json response uses "nakk"

Reply:
These should be in the "OperatorStatisticNames" dict, "OperatorStatistics" is deprecated. Other than that, this looks good.